environment-wasm-host.yml

```diff
@@ -1,20 +1,19 @@
 name: xeus-r-wasm-host
 channels:
-  - https://repo.mamba.pm/emscripten-forge
-  - https://repo.mamba.pm/conda-forge
+  - https://repo.prefix.dev/emscripten-forge-dev
+  - conda-forge
 dependencies:
   # Host dependencies
-  - emscripten-abi==3.1.45
+  - emscripten-abi==3.1.73
```

Comment:
I remember one of Martin's reviews on my PR while moving to 3..1.73 for xeus-cpp that the `emscripten-abi` dependency can completely be removed. I think the same should work here too !

Reply:
True, it's not needed since the packages are hosted in different channels